assets/src/scss/new-identity/style.scss

```diff
@@ -1,14 +1,26 @@
 @import "../base/tokens";
+@import "../base/variables";
+@import "../base/mixins";
 
 @mixin shared-link-styles {
+  --link--height: 26px;
+  --transition--duration: 0.3s;
+
   border-bottom: 2px solid var(--gp-green-500);
-  transition: 0.3s cubic-bezier(0.5, 0, 0, 1);
+  transition: var(--transition--duration) cubic-bezier(0.5, 0, 0, 1);
+
+  &:hover {
+    box-shadow: inset 0 calc(var(--link--height) * -1) rgba($gp-green-500, .15);
+  }
 
   &:hover,
-  &:focus {
-    box-shadow: inset 0 calc(-22px) var(--gp-green-100);
+  &:active {
     text-decoration: none !important;
```

Comment:
Maybe if we remove [these lines](https://github.com/greenpeace/planet4-plugin-gutenberg-blocks/blob/main/assets/src/styles/blocks/SplitTwoColumns.scss#L155-L159) (useless anyways because these links follow body links styles) then the `!important` is no longer needed?

Reply:
To keep the current functionality still working as the UI expected, I would prefer to use a `CSS variable` 
```css
  &:hover {
    text-decoration: var(--link--hover--text-decoration, underline);
  }
```
and then change that local variable through the new-identity stylesheet
```css
  &:hover,
  &:active {
    --link--hover--text-decoration: none;
  }
```

Does it make sense?

Reply:
See this [other (and related) PR](https://github.com/greenpeace/planet4-plugin-gutenberg-blocks/pull/1045)